src/App.tsx

```diff
@@ -16,47 +16,93 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [
+    sortedGoods,
+    setSortedGoods,
+  ] = useState(goodsFromServer);
+  const [isVisibleReser, setIsVisibleReser] = useState(false);
+
+  enum SortType {
+    Alphabet = 'alphabet',
+    Length = 'length',
+    Reverse = 'reverse',
+  }
+
+  function handleSort(filter: string) {
```

Comment:
You are not filtering, but sorting the list, why is the argument called a `filter`?